state/redis/metadata.yaml

```
@@ -67,10 +69,17 @@ metadata:
     example: "30s"
     type: duration
   - name: redisType
+    type: string
     required: false
-    description: The type of redis. There are two valid values, one is \"node\" for single node mode, the other is \"cluster\" for redis cluster mode. Defaults to \"node\".
+    allowedValues:
+      - "node"
+      - "cluster"
+    default: "node"
+    description: |
+      The type of redis. There are two valid values, one is "node" for single node mode, the other is
+      "cluster" for redis cluster mode.
+      Defaults to "node".
```

```suggestion
      Redis service type. Set to "node" for single-node mode, or "cluster" for Redis Cluster.
```

No need to write the default mode since it's included below.